README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
If you run my repo locally, make sure you're against current master as I have some changes to hopefully get the codeclimate run to complete.  Hopefully.  🙏